src/java.base/share/classes/javax/net/ssl/SSLEngine.java

```diff
@@ -673,6 +673,9 @@ public abstract SSLEngineResult wrap(ByteBuffer [] srcs, int offset,
      * @throws  IllegalStateException if the client/server mode
      *          has not yet been set.
      * @see     #unwrap(ByteBuffer, ByteBuffer[], int, int)
+     *
+     * @implNote The data in {@code src} may be modified during the decryption
```

Comment:
It looks like a note for the API users to me.  Is apiNote tag more appropriate here?

Reply:
The CSR and this code is changing, the note I was adding was already documented, but its existing wording should be more clear.

Reply:
I like the new update as specified in the CSR.  I added my name as Reviewer for the CSR.